cfme/tests/automate/test_common_methods.py

```diff
@@ -180,3 +182,53 @@ def test_miq_password_decrypt(klass):
         execute_methods=True,
     )
     assert result.validate()
+
+
+@pytest.fixture(scope="module")
+def vm_folder(provider):
+    """Create Vm folder on VMWare provider"""
+    folder = provider.mgmt.create_folder(f"{fauxfactory.gen_alpha()}")
+    yield folder
+    vm_folder.Destroy()
```

Comment:
are you sure there should be vm_folder instead of folder ?

Reply:
My bad!
It should be folder.
Thanks :)